train-tracker/src/Search.js

```diff
@@ -69,10 +86,16 @@ function Search({searchFun, routes, stations}){
                     </span>
                 </label>
 
+                <label className="favorites-dropdown-selection">
```

Comment:
honestly wouldn't be opposed to putting the favorites dropdown before the list, possibly in it's own div

Reply:
Seconding this, lets not clutter the search div too much, and this is a neat functionality anyways.